meson_options.txt

```diff
@@ -0,0 +1,8 @@
+option('exportable-egl',
+	type: 'boolean',
+	value: true,
+	description: 'Enable the exportable EGL interface')
```

Comment:
I think this option should be abandoned. I wouldn't add it to Meson, unless we'd want it for compatibility with CMake. Wherever it is, it shouldn't do anything, and the EGL-based exportable should be built and installed unconditionally.

Reply:
Done in the latest iteration of the patch set :ok_hand: